### ISSUE TITLE
Loss Difference between paper and code

### ISSUE THREAD

**Issue by RickyYXY** [duanyiqun/DiffusionDepth#6]
I notice that you have mentioned a latent space loss in the paper(equ.7).
But I can’t find this loss in the training code.
So which one is right? This is strange.

**Comment by duanyiqun** [comment]
Hi Ricky, we found after the paper's formation that, the latent space loss only boosts slightly higher performance but with a much higher computational cost. So we remove the public version. 
Regards

**Comment by RickyYXY** [comment]
Get it! And I find that the depth pixel loss is directly L1 L2 loss, not like loss in Equ.6. Is it because using Equ.6 makes no difference? Thanks for replying me so fast.

**Comment by RickyYXY** [comment]
OK, so if I want the best performance, I should change the default config and use the SIGloss instead of L1+L2 loss, right?

**Comment by duanyiqun** [comment]
I would suggest for SIGLoss + L1+L2 + DDIM loss for initialization and SIGLoss + DDIM for finetune.

**Comment by RickyYXY** [comment]
OK! Thanks for your reply! Helped me a lot!
